### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by Sameony**
Created snackbar/toast component. Two types: normal and stacked.
image

### CODE REVIEW

**Review by SimranjitKaurMaan — APPROVED** [review]

File: Templates/Snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  background-color: var(--dark-gray);
+  max-width: 50rem;
+  color: var(--snow);
+  border-radius: 5px;
+  box-shadow: 6px 6px 9px var(--gray);
+}
+.normalSnack {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (sadathshariff) [inline Templates/Snackbar/snackbar.css:11]: You can make this a utility class and use that

**Review by sadathshariff — COMMENTED** [review]
Everything looks good

File: Templates/Snackbar/snackbar.css (hunk 1 above)
Comment (Sameony) [reply Templates/Snackbar/snackbar.css:11]: yes actually, I thought of using lesser utility classes to keep the html cleaner. The justify and align properties are something which I hardly use in my current layout(I use space around sometimes, center other times, etc.), so I thought of avoiding making it into a utility class. If I do encounter more such cases, I will make utility class for it. Thanks for pointing it out.

**Review by sadathshariff — APPROVED** [review]

File: Templates/Snackbar/snackbar.css (hunk 1 above)
Comment (sadathshariff) [reply Templates/Snackbar/snackbar.css:11]: Even I'm using fewer utility classes for the same reason but yeah if you don't have a separate utility class it's fine

File: Styles/main.css [hunk 2]
```diff
@@ -6,6 +6,7 @@
 @import url(/Templates/Inputs/inputs.css);
 @import url(/Templates/Cards/cards.css);
 @import url(/Templates//Ratings/ratings.css);
+@import url(/Templates/Snackbar/snackbar.css);
 
 :root {
```
Comment (vaibhav-mahalle) [inline Styles/main.css]: No need of specifying font size is 16px by default

File: Templates/Snackbar/snackbar.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  cursor: pointer;
+}
+.cross {
+  cursor: pointer;
+  transition: ease-in 0.5s;
+}
+.cross:hover {
+  background-color: var(--dark-gray);
+  border-radius: 5px;
+}
```
Comment (vaibhav-mahalle) [inline Templates/Snackbar/snackbar.css:38]: Good structured code without clutter.

**Review by vaibhav-mahalle — APPROVED** [review]
Clean code. I like that you have used came casing for variable names.

File: Styles/main.css (hunk 2 above)
Comment (Sameony) [reply Styles/main.css]: ah, right, i'll remove it . Was just experimenting with different sizes and rems

**Pull request merged**